contrib/seeds/nodes_main_manual.txt

```diff
@@ -510,518 +510,6 @@ zvchlrjuzqdlx37fhibhnym4y6p56vtlymujjuzhh2cp34yqfrtq.b32.i2p:0
 zxsd3fqczh6ddgejc24nnmb3ww7nalieq3a7cs2mqiy6tmff3wia.b32.i2p:0
 zy2ywvyqds5bgdoo4tgbu3bwjp3ygyn3zfuby44jemc6xa6fbwta.b32.i2p:0
 zzre44vh766jgfordw2ehu2r6p44j23uyovgvm7iwuhp3g5iz4ca.b32.i2p:0
-ycdw2e4ufgfwhcqna4g3m2qsvaly23ozaexawcj3x4gtgcehgwujjgid.onion:8333
```

Comment:
They were updated in #29561, and the addresses were pulled from my node's addrman. Some sorting happened somewhere, and because `makeseeds.py` doesn't shuffle (this PR adds a commit that does that), when it applied the max node count, it ended up with the tail end of that list.